src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page/components/App/index.js

```diff
@@ -1,57 +1,161 @@
 // Node modules.
 import React from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import { isEmpty } from 'lodash';
-import { selectPatientFacilities } from '@department-of-veterans-affairs/platform-user/cerner-dsot/selectors';
+import CallToActionWidget from 'applications/static-pages/cta-widget';
 // Relative imports.
-import { selectEhrDataByVhaId } from 'platform/site-wide/drupal-static-data/source-files/vamc-ehr/selectors';
-import AuthContent from '../AuthContent';
-import UnauthContent from '../UnauthContent';
-import {
-  ehrDataByVhaIdPropType,
-  facilitiesPropType,
-  useSingleLogoutPropType,
-} from '../../../propTypes';
+import MoreInfoAboutBenefits from '../../../components/MoreInfoAboutBenefits';
 
-export const App = ({
-  ehrDataByVhaId,
-  facilities,
-  useSingleLogout,
-  widgetType,
-}) => {
-  const cernerFacilities = facilities?.filter(f => f.usesCernerAppointments);
-  const otherFacilities = facilities?.filter(f => !f.usesCernerAppointments);
-  if (!isEmpty(cernerFacilities)) {
-    return (
-      <AuthContent
-        cernerFacilities={cernerFacilities}
-        otherFacilities={otherFacilities}
-        ehrDataByVhaId={ehrDataByVhaId}
-        useSingleLogout={useSingleLogout}
-        widgetType={widgetType}
-      />
-    );
-  }
-
-  return <UnauthContent />;
-};
+export const App = () => (
+  <>
```

Comment:
Copied this from the (misnamed) `UnauthContent` widget, which had the CallToActionWidget we wanted